.github/workflows/coverity.yml

```diff
@@ -0,0 +1,61 @@
+name: Coverity scan build
+
+on:
+  schedule:
+    - cron: '30 2 * * 6'
```

Comment:
I think we can do it even every day. The FAQ list this:
> * Up to 28 builds per week, with a maximum of 4 builds per day, for projects with fewer than 100K lines of code
> * Up to 21 builds per week, with a maximum of 3 builds per day, for projects with 100K to 500K lines of code
> * Up to 14 builds per week, with a maximum of 2 build per day, for projects with 500K to 1 million lines of code
> * Up to 7 builds per week, with a maximum of 1 build per day, for projects with more than 1 million lines of code

https://scan.coverity.com/faq#frequency

Reply:
Initially I had `cron: '30 2 * * 0,2,4'`, so doing 3 builds a week, but it seemed too much, so I went with a current weekly build. We can fine tune this later if we find out, that a weekly scan is not good enough.

Instead of increasing a cadence of snapshot scans, I would probably start thinking about scanning of stable releases. Old stable perhaps monthly, stable bi-weekly?

Reply:
scanning stable releases is good.
Currently new scans overwrite the old ones, does coverity  support multiple branches?

Reply:
>  Currently new scans overwrite the old ones, does coverity support multiple branches?

I don't think so, I was thinking about having separate project for each stable release, thus `https://scan.coverity.com/projects/openwrt-22.03`. Yes, it's cumbersome for setup/management/users, but I don't see better solution.